include/measurement_kit/neubot/dash.hpp

```diff
@@ -0,0 +1,26 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NEUBOT_DASH_HPP
+#define MEASUREMENT_KIT_NEUBOT_DASH_HPP
+
+#include <functional>
+#include <measurement_kit/common.hpp>
+#include <measurement_kit/ext.hpp>
+
+namespace mk {
+namespace neubot {
+namespace dash {
+
+// Run a dash test
+void run(Settings settings, Callback<Error,  Var<nlohmann::json>> cb,
```

Comment:
Also, rather than exposing `nlohmann::json`, could you see whether the code would work exposing `report::Entry`? If it does not work, let me know why, and I'll see if I could quickly patch Entry up.